test/cookie.js

```diff
@@ -39,6 +39,17 @@ describe('new Cookie()', function () {
       assert.throws(function () { new Cookie(function () {}) }, /argument options/)
     })
 
+    it('should accept integer maxAge options', function () {
+      var cookie = new Cookie({ maxAge: 7200000 })
+		  assert.equal(typeof cookie.maxAge, 'number')
+		  assert.equal(cookie.maxAge % 1, 0)
+	  })
```

Comment:
There is definitely something wrong with either eslint for not catching this or Github is not displaying this right. This closing brace should be aligned with the "it"